tools/inspect/deprecated_include_check.cpp

```diff
@@ -51,7 +52,7 @@ namespace boost { namespace inspect {
         {"boost/format\\.hpp", "pika/util/format.hpp"},
         {"boost/regex.hpp", "regex"},
         {"boost/program_options([^\\s]*)\\.hpp", "pika/program_options\\2.hpp"},
-        {"boost/filesystem([^\\s]*)\\.hpp", "pika/modules/filesystem.hpp"},
+        {"boost/filesystem([^\\s]*)\\.hpp", "filesystem"},
```

Comment:
as `boost/filesystem.hpp` was already deprecated, shouldn't the module header be deprecated instead?
```suggestion
        {"pika/modules/filesystem.hpp", "filesystem"},
```

Reply:
The filesystem module and `pika/modules/filesystem.hpp` still exist because not all of the functionality exists in `<filesystem>`. `<filesystem>` is to be used whenever possible, but `pika/modules/filesystem.hpp` is not deprecated.